javascriptExercises/page232-270Exercises/public/the-cat-hat.css

```diff
@@ -0,0 +1,19 @@
+div{
```

Comment:
Please contain all files that relate to one exercise in a single folder. Always try to think how the other developer reviewing the code will look at things. For example I need to jump between different folders just to understand how things are implemented